packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -177,7 +178,7 @@ export class OdspDeltaStorageWithCache implements IDocumentDeltaStorageService {
 				validateMessages("cached", messages, from, this.logger);
 				if (messages.length > 0 && messages[0].sequenceNumber === from) {
 					this.snapshotOps = this.snapshotOps.filter((op) => op.sequenceNumber >= to);
-					opsFromSnapshot = messages.length;
+					opsFromSnapshot += messages.length;
```

Comment:
Could this be related to the times we see NoJoinOps in ODSP tests? Gary confirmed from their end it looks like the sequenced join op was sent by the server within 100ms. I know the PR and the comments in the ADO item are about FRS, but seeing this change here made me wonder if it was a potential issue for ODSP too.

Reply:
I'm not sure if this change is material, given this.snapshotOps = undefined below. Does it change anything?

Reply:
No @alexvy86 , that opsFromSnapshot var is used only for telemetry purposes.
@vladsud This is same logic which I have copied to frs driver. In frs driver, on fetch of first 2K ops, we used to set the snapshot ops as undefined even though it could have more than 2K ops. With this change, we dump the ops that we have used and keep the rest so that they can be used further. So only unused ops are kept. It could impact the no join op issue in frs driver along with increasing the MaxBatchDeltas to 5K.